### PR TITLE
4703 - Don't require WixBalExtension when using WixNetFxExtension

### PR DESCRIPTION
**Pull request by rseanhall**

### CODE REVIEW

File: src/chm/documents/customactions/wixnetfxextension.html.md [hunk 1]
```diff
@@ -7,122 +7,6 @@ after: using_standard_customactions
 
 The [WixNetfxExtension](~/xsd/netfx/index.html) includes a set of custom actions to compile native images using Ngen.exe. For an example, see [How To: NGen managed assemblies during installation](~/howtos/files_and_registry/ngen_managed_assemblies.html).
 
-## PackageGroups
```
Comment (robmen) [inline src/chm/documents/customactions/wixnetfxextension.html.md:10]: @barnson good idea to move all these?

File: src/ext/BalExtension/wixlib/MbaPrereqs.wxs [hunk 2]
```diff
@@ -0,0 +1,168 @@
+<?xml version='1.0' encoding='utf-8'?>
```
Comment (robmen) [inline src/ext/BalExtension/wixlib/MbaPrereqs.wxs]: Okay, but should we break this out into separate files? I thought @barnson did that when the Nefx.wxs got too crazy with everything in one file. Thoughts?

File: src/chm/documents/customactions/wixnetfxextension.html.md (hunk 1 above)
Comment (barnson) [inline src/chm/documents/customactions/wixnetfxextension.html.md:10]: I think reference material like this should stay with its "owning" topic. How-tos are less discoverable.

File: src/ext/BalExtension/wixlib/MbaPrereqs.wxs (hunk 2 above)
Comment (rseanhall) [inline src/ext/BalExtension/wixlib/MbaPrereqs.wxs]: I'm neutral on this, I can break them out.

File: src/chm/documents/customactions/wixnetfxextension.html.md (hunk 1 above)
Comment (rseanhall) [inline src/chm/documents/customactions/wixnetfxextension.html.md:10]: I think I should have linked to the how-to from here.  It doesn't make sense to me for bundle information like this to be under the `customactions` folder, I don't think that's discoverable either.  Maybe create a NetFxExtension page under the `bundles` folder for this information?

Comment (barnson) [inline src/chm/documents/customactions/wixnetfxextension.html.md:10]: Yeah, that's a bad name. The content is about the extensions and everything they contain. Probably an artifact of people finding a place that mostly fits. :)

**Pull request merged**